extension.driver.php

```diff
@@ -305,38 +305,44 @@ private function processFields(XMLElement $xml)
             foreach ($elements as $element_index => $element) {
 
                 // get element handle
+                if (is_object($element)) {
```

Comment:
Could you change this line for this:

    if ($element instanceof XMLElement) {

like used in the `findEntries()` function above?